src/grocker/__main__.py

```diff
@@ -105,13 +106,18 @@ def __init__(self, option_strings, *args, **kwargs):
         super(PurgeAction, self).__init__(
             option_strings=option_strings,
             dest=argparse.SUPPRESS,
-            choices=('all', 'builders', 'dangling', 'runners'),
+            choices=('old', 'old:runner', 'all', 'all:runner'),
         )
 
     def __call__(self, parser, namespace, values, *args, **kwargs):
-        filters = [values] if values != 'all' else [x for x in self.choices if x != 'all']
-        builders.docker_purge_images(builders.docker_get_client(), filters)
-        builders.docker_purge_volumes(builders.docker_get_client(), filters)
+        current_version = 'all' in values
+        runner = 'runner' in values
+
+        loggers.setup()
+        docker_client = utils.docker_get_client()
+        cleanners.docker_purge_container(docker_client, current_version=current_version)
+        cleanners.docker_purge_volumes(docker_client, current_version=current_version)
+        cleanners.docker_purge_images(docker_client, current_version=current_version, runner=runner)
```

Comment:
Now that I understand where `PurgeAction` is dealt with, this makes me sad.
`grocker --purge runners --help` should not be doing anything.